packages/lb-components/src/views/MainView/toolFooter/PageNumber/index.tsx

```diff
@@ -30,7 +30,19 @@ const PageNumber = (props: IProps) => {
     return null;
   }
 
-  const count = toolInstance?.currentPageResult?.length ?? (globalPattern === EPointCloudPattern.Detection ? pointCloudBoxList.length : segmentation.length);
+  /**
+   * Count Showing Priority.
+   * 
+   * 1. currentPageCount
+   * 2. currentPageResult?.length.
+   * 3. PointCloud data.
+   */
+  const count =
+    toolInstance?.currentPageCount ??
+    toolInstance?.currentPageResult?.length ??
+    (globalPattern === EPointCloudPattern.Detection
```

Comment:
add currentPageCount on pointcloud instance

Reply:
It is a good idea!